### PR TITLE
Spraypaint

### PR DESCRIPTION
**Pull request by mjschock**
## what:

* Hide brand in the footer toggle (STENCIL-2938)
* Sale badges (STENCIL-2640)
* Resizing of swatches (STENCIL-2540)
* Require login for pricing (STENCIL-2578)
* As low as display (MERC-1165)
* Social media placement (STENCIL-2530)
* UX adjustments (STENCIL-2670)

### CODE REVIEW

File: assets/scss/layouts/products/_productSwatch.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// =============================================================================
+// SWATCH SIZES (CSS)
+// =============================================================================
+
+
+// Swatch Size Changes
+// -----------------------------------------------------------------------------
+
+// do some math
+
```
Comment (bc-ong) [inline assets/scss/layouts/products/_productSwatch.scss]: Extra whitespace.

File: assets/scss/layouts/products/_productSaleBadges.scss [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+// BURST BADGE
+// -----------------------------------------------------------------------------
+.sale-flag-star,
+.sale-flag-star:before,
+.sale-flag-star:after {
+    background: stencilColor("color_badge_product_sale_badges");
+    content: "";
+    display: block;
+    float: none;
```
Comment (mjschock) [inline assets/scss/layouts/products/_productSaleBadges.scss]: the float has been removed

File: assets/scss/layouts/products/_productSaleBadges.scss [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+}
+
+.starwrap {
+    height: rem-calc(50px);
+    position: absolute;
+    top: 10px;
+    width: rem-calc(50px);
+}
+
+span.sale-text-burst {
```
Comment (mjschock) [inline assets/scss/layouts/products/_productSaleBadges.scss]: it's only one element, the span has been removed

File: assets/scss/layouts/products/_productSaleBadges.scss [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+    color: stencilColor("color_text_product_sale_badges");
+    font-weight: 600;
+    line-height: rem-calc(15px);
+    position: absolute;
+    text-align: center;
```
Comment (mjschock) [inline assets/scss/layouts/products/_productSaleBadges.scss]: yes, it appears so.  the span has been replaced with a div.

File: assets/scss/layouts/products/_productSaleBadges.scss [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+    top: 20%;
+    width: rem-calc(50px);
```
Comment (mjschock) [inline assets/scss/layouts/products/_productSaleBadges.scss]: using a div instead.

File: assets/scss/layouts/products/_productSaleBadges.scss [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+// -----------------------------------------------------------------------------
+// TOP LEFT BADGE
+// -----------------------------------------------------------------------------
+
+.sale-flag-side {
+    background: stencilColor("color_badge_product_sale_badges");
+    border-radius: 0 50px 50px 0;
+    color: stencilColor("color_text_product_sale_badges");
+    float: left;
```
Comment (mjschock) [inline assets/scss/layouts/products/_productSaleBadges.scss]: the float has been removed

File: assets/scss/layouts/products/_productSaleBadges.scss [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+}
+
+// -----------------------------------------------------------------------------
+// SASH BADGE
+// -----------------------------------------------------------------------------
+
+.sale-flag-sash {
+    background: stencilColor("color_badge_product_sale_badges");
+    color: stencilColor("color_text_product_sale_badges");
+    float: left;
```
Comment (mjschock) [inline assets/scss/layouts/products/_productSaleBadges.scss]: float removed

**Comment by bc-ong** [comment]
:+1:

**Pull request merged**